src/Components/Users/UserProfile.tsx

```diff
@@ -165,14 +166,12 @@ export default function UserProfile() {
             setIsLoading(true);
             const data = {
                 username: username,
-                first_name: states.form.first_name,
-                last_name: states.form.last_name,
+                first_name: states.form.firstName,
+                last_name: states.form.lastName,
                 email: states.form.email,
-                phone_number: parsePhoneNumberFromString(states.form.phone_number)?.format('E.164'),
+                phone_number: parsePhoneNumberFromString(states.form.phoneNumber)?.format('E.164'),
```

Comment:
Codacy found an issue: [Identifier 'phone_number' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5838477)